libexec/rtld-elf/rtld_paths.h

```diff
@@ -91,8 +91,7 @@
 
 #ifndef STANDARD_LIBRARY_PATH
 #ifdef RTLD_SANDBOX
-#define	STANDARD_LIBRARY_PATH	"/lib" _RTLD_COMPAT_LIB_SUFFIX "/casper:" \
-				"/usr/lib" _RTLD_COMPAT_LIB_SUFFIX "/c18n:" \
+#define	STANDARD_LIBRARY_PATH	"/usr/lib" _RTLD_COMPAT_LIB_SUFFIX "/c18n:" \
```

Comment:
This makes me think: as we move towards always-on c18n, we're technically going to need c18n libc to be in /lib/c18n for the same reason non-c18n is in /lib.